example_tests/complex_checks/inline_body_schema.jkt

```diff
@@ -1,5 +1,5 @@
 name: Inline Body Schema
-tags: regression flaky triaged
+tags: fuzzy_check
```

Comment:
I'd probably simplify this tag name. Is this just an example of "fuzzing"? We could just label it as "fuzz". I'd be hard pressed to want "multi-word" tags if they can be avoided and simplified.

Reply:
its an example of using a schema for verification instead of providing a literal value. I can rename tag to one of bodySchema, schema, or fuzzing ( whichever we feel is most appropriate ).